src/Libraries/CoreNodeModels/Input/File.cs

```diff
@@ -321,8 +321,8 @@ public override void Dispose()
         }
     }
 
-    [NodeName("Directory.FromPath")]
-    [NodeCategory(BuiltinNodeCategories.CORE_IO)]
+    [NodeName("DirectoryFromPath")]
+    [NodeCategory("Core.File")]
```

Comment:
@aparajit-pratap This node in `CoreNodeModels` is defined as follows in a DYN file: 
```
<CoreNodeModels.Input.DirectoryObject type="CoreNodeModels.Input.DirectoryObject" nickname="Directory.FromPath" ...> ...
    </CoreNodeModels.Input.DirectoryObject>
```
When a file containing this node is opened, the node still functions as `CoreNodeModels.Input.DirectoryObject`, but its name remains as `Directory.FromPath`.